lib/active_merchant/billing/gateways/worldpay.rb

```diff
@@ -270,6 +270,16 @@ def add_payment_method(xml, amount, payment_method, options)
               xml.tag! 'session', 'shopperIPAddress' => options[:ip] if options[:ip]
               xml.tag! 'session', 'id' => options[:session_id] if options[:session_id]
             end
+
+            if three_d_secure = options[:three_d_secure]
+              xml.tag! 'info3DSecure' do
+                xml.tag! 'threeDSVersion', three_d_secure[:version]
```

Comment:
doc states:
```
threeDSVersion
Possible Values:
1.0.2
2.1.0
```
how do we plan to handle this? The version passed in the options might be different, say `2.2.0`

Reply:
I wonder what really matters, if it's more like 1 vs. 2, or the actual number matters

Reply:
I'm confirming this with Worldpay, but rather not block this PR.